base/src/middleware/http_errors.mustache

```diff
@@ -0,0 +1,18 @@
+const HttpError = require('standard-http-error');
+const {
+  thinky: {
+    Errors: { DocumentNotFound, ValidationError }
+  }
+} = require('../../config');
+
+module.exports = (err, req, res, next) => {
+  if (err instanceof DocumentNotFound) {
+    res.status(404).send({ error: 'not found' });
```

Comment:
Yep, fair point. I don't think it's a big deal either way, just twigged me as I was reading it over.

The time I can see it being useful in a constants file is if you're checking for the error in an integration test.

This comment doesn't block merging this PR. It's not a severe issue.